src/pint/utils.py

```diff
@@ -299,6 +300,52 @@ def split_prefixed_name(name):
     return prefix_part, index_part, int(index_part)
 
 
+def get_param_name_map(components):
+    """ Get the builting parameter name mapping, including the aliases and the
+    prefix mapping.
```

Comment:
Docstrings should start with a one-line description, with no space in front of it.

Also can you explain what this does? What does this map to what?